lib/formatters/html.js

```diff
@@ -44,45 +48,77 @@ handlebars.registerHelper("renderText", function(totalErrors, totalWarnings) {
  * @param {string} totalWarnings Total warnings
  * @returns {int} The color code (0 = green, 1 = yellow, 2 = red)
  */
-handlebars.registerHelper("getColor", function(totalErrors, totalWarnings) {
+function renderColor(totalErrors, totalWarnings) {
     if (totalErrors !== 0) {
         return 2;
     } else if (totalWarnings !== 0) {
         return 1;
     }
     return 0;
-});
+}
 
 /**
- * Get the HTML row content based on the severity of the message
- * @param {int} severity Severity of the message
- * @returns {string} The generated HTML row
+ * @param {Array} messages Messages.
+ * @param {number} parentIndex Index of the parent HTML row.
+ * @returns {string} HTML string describing result messages.
  */
-handlebars.registerHelper("getSeverity", function(severity) {
-    // Return warning else error
-    return new handlebars.SafeString((severity === 1) ? "<td class=\"clr-1\">Warning</td>" : "<td class=\"clr-2\">Error</td>");
-});
+function renderMessages(messages, parentIndex) {
+    return lodash.map(messages, function(message) {
+        var lineNumber,
+            columnNumber;
+
+        lineNumber = message.line || 0;
+        columnNumber = message.column || 0;
+
+        return messageTemplate({
+            parentIndex: parentIndex,
+            lineNumber: lineNumber,
+            columnNumber: columnNumber,
+            severityNumber: message.severity,
+            severityName: message.severity === 1 ? "Warning" : "Error",
+            message: message.message,
+            ruleId: message.ruleId
+        });
+    }).join("\n");
+}
+
+/**
```

Comment:
function description missing as we try to put descriptions in jsdoc for each function.
Yes, our static analysis should check this but we are working on it to activate more rules for all the eslint repos.